modules/FindFreetype.cmake

```diff
@@ -145,10 +172,9 @@ if(FREETYPE_INCLUDE_DIR_freetype2 AND FREETYPE_H)
   endforeach()
 endif()
 
-
 # handle the QUIETLY and REQUIRED arguments and set FREETYPE_FOUND to TRUE if
 # all listed variables are TRUE
-include(${CMAKE_CURRENT_LIST_DIR}/FindPackageHandleStandardArgs.cmake)
+include(${CMAKE_ROOT}/Modules/FindPackageHandleStandardArgs.cmake)
```

Comment:
Oops, I just made that change without testing and actually it doesn't work for `SelectLibraryConfigurations.cmake` so that one will have to remain absolute.